tests/test_metrics/test_box_overlap.py

```diff
@@ -21,7 +21,8 @@ def _construct_bbox(num_bbox=None):
         bboxes[:, 0::2] *= img_w
         bboxes[:, 1::2] *= img_h
         return bboxes, num_bbox
-
+    
+    # test mode 'giou'
```

Comment:
There are some modifications forgot to delete. You can directly copy the original file to cover it.

Reply:
ok.